code/script.js

```diff
@@ -230,96 +323,176 @@ const setSecret = () => {
 // This function to start (and restart) the game
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
-  charactersInPlay = CHARACTERS
+  charactersInPlay = MUSHROOMS
+  board.style.display = 'flex';
```

Comment:
so you change the css of the board from here... I see you got the variable from the html, but I cannot find it in the HTML. Is it needed?